src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+
+Today, we are releasing a major update to the app with more codecs support, a new design, and a new way to use Squoosh on your command line called Squoosh CLI.
+
+
+## New codecs support
+
+We now support OxiPNG, MozJPEG, WebP, and AVIF, in addition to codecs natively supported in your Browser.  A new codec was made possible again with the use of WebAssembly. By compiling a codec encoder and decoder as WebAssembly module users can access and experiment with newer codecs even if their preferred browser does not support them. 
+
+To learn more about different codecs, we are kicking off Understanding Image Codecs series on web.dev so stay tuned!
+
+
+## Launching a command line Squoosh!
+
+Ever since the original launch in 2018, common user request was to interact with Squoosh programmatically without UI. We felt a bit conflicted about this path since our app was a UI on top of command line based codec tools. However we do understand the desire to interact with the whole package of codecs instead of multiple tools. Squoosh CLI does just that.
+
+{% YouTube 'FUqn8eOxCP4' %}
+
+You can install the beta version of the Squoosh CLI by running `npm -i @squoosh/cli` or run it directly using `npx @squoosh/cli [parameters]`.
+
+The Squoosh CLI is written in Node and makes use of the exact same WebAssembly modules the PWA uses. Through extensive use of Workers, all images are decoded, processed and encoded in parallel. We also use Rollup to bundle everything into one JavaScript file to make sure installation via `npx` is quick and seamless. The CLI also offers auto compression, where it tries to reduce the quality of an image as much as possible without degrading the visual fidelity (using the [Butteraugli metric](https://github.com/google/butteraugli)). 
+
+With the Squoosh CLI you can compress the images in your web app to multiple formats and use the [`<picture>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#The_type_attribute) to let the browser choose the best version. We also plan to build plugins for Webpack, Rollup etc to make image compression a part of your build step that happens automatically.
+
+
+## Build process change from Webpack to Rollup
+
+The same team that built Squoosh has spent a significant amount of time looking at build tooling this year for [Tooling Report](https://bundlers.tooling.report/), and decided to switch our build process from webpack to rollup.
+
+The project was initially started with a webpack because we wanted to try it as a team, and at the time in 2018 webpack was the only tool that gave us enough control to set up the project the way we wanted. Over time, we've found Rollup's ease of plugin systeme and simplicity with ESM made it a natural choice for this project. 
```

Comment:
```suggestion
The project was initially started with webpack because we wanted to try it as a team, and at the time in 2018 webpack was the only tool that gave us enough control to set up the project the way we wanted. Over time, we've found Rollup's easy plugin system and simplicity with ESM made it a natural choice for this project. 
```